src/jit/lir.h

```diff
@@ -282,6 +282,10 @@ class LIR final
         Range Remove(GenTree* firstNode, GenTree* lastNode);
         Range Remove(ReadOnlyRange&& range);
 
+        void Delete(GenTree* node, BasicBlock* block, Compiler* compiler);
+        void Delete(GenTree* firstNode, GenTree* lastNode, BasicBlock* block, Compiler* compiler);
+        void Delete(ReadOnlyRange&& range, BasicBlock* block, Compiler* compiler);
+
```

Comment:
Yeah, the ordering is an artifact of the original changes, which had default values of `nullptr` for `block` and `compiler`. I'll make the change (which is consistent with the ordering for the removed `DecRefCnts` method).